### ISSUE TITLE
Aside should not be contained in another landmark

### ISSUE THREAD

**Issue by JuliaHempel** [zopaUK/react-components#684]
In the [`<Navbar>`](https://github.com/zopaUK/react-components/blob/master/src/components/organisms/Navbar/Navbar/Navbar.tsx) we but an `<aside>` element inside `<nav>` element. 

[`<nav>`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Navigation_Role) is a landmark element 
[`<aside>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside) is a landmark element with a complementary ARIA role

Complementary landmarks [should be top level landmarks](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/complementary.html#:~:text=complementary%20landmarks%20should%20be%20top,should%20have%20a%20unique%20label.)
